concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-309	309	Ladro	thief	1160	THIEF	605	629	571	263	650	121	375
+DellaRosa-2010-417-310	310	Lago	lake	624	LAKE	676	679	553	240	633	119	215
+DellaRosa-2010-417-311	311	Lampada	lamp	625	LAMP	700	700	665	197	674	117	145
+DellaRosa-2010-417-312	312	Latte	milk	635	MILK	695	662	674	138	675	103	181
+DellaRosa-2010-417-313	313	Lavoratore	worker	2977	WORKER	618	538	587	315	621	162	374
+DellaRosa-2010-417-314	314	Leone	lion	1386	LION	681	687	553	178	648	133	155
+DellaRosa-2010-417-315	315	Letto	bed	1663	BED	697	693	676	166	683	113	110
+DellaRosa-2010-417-316	316	Libreria	bookcase			688	685	660	275	638	110	197
+DellaRosa-2010-417-317	317	Limone	lemon	3091	LEMON	695	691	653	176	668	106	288
+DellaRosa-2010-417-318	318	Madre	mother	1216	MOTHER	668	688	700	136	685	156	275
+DellaRosa-2010-417-319	319	Maiale	pig	1337	PIG	700	697	593	215	642	100	185
+DellaRosa-2010-417-320	320	Malattia	disease	1986	DISEASE	497	487	618	219	609	263	270
+DellaRosa-2010-417-321	321	Marea	tide	891	TIDE	535	561	416	413	579	221	419
+DellaRosa-2010-417-322	322	Martello	hammer	1481	HAMMER	700	697	595	209	653	106	275
+DellaRosa-2010-417-323	323	Matita	pencil	2578	PENCIL	694	684	679	194	694	100	121
+DellaRosa-2010-417-324	324	Matrimonio	wedding	3182	MARRIAGE OR WEDDING	489	621	616	248	632	313	375
+DellaRosa-2010-417-325	325	Mela	apple	1320	APPLE	694	676	670	156	694	100	121
+DellaRosa-2010-417-326	326	Melanzana	eggplant	1146	AUBERGINE	700	695	545	275	600	100	208
+DellaRosa-2010-417-327	327	Mendicante	beggar	1934	BEGGAR	637	629	405	333	574	163	300
+DellaRosa-2010-417-328	328	Minerale	mineral			627	538	447	396	505	174	388
+DellaRosa-2010-417-329	329	Mucca	cow	1007	COW	691	662	618	163	680	100	206
+DellaRosa-2010-417-330	330	Muscolo	muscle	1004	MUSCLE	623	635	616	344	588	156	319
+DellaRosa-2010-417-331	331	Musica	music	642	MUSIC	473	459	676	188	630	291	200
+DellaRosa-2010-417-332	332	Naso	nose	1221	NOSE	688	680	658	141	688	100	116
+DellaRosa-2010-417-333	333	Negozio	shop	752	STORE	668	648	658	221	656	125	163
+DellaRosa-2010-417-334	334	Noce	walnut	2042	WALNUT	694	668	591	235	638	105	147
+DellaRosa-2010-417-335	335	Nomade	nomad			588	548	339	418	544	190	416
+DellaRosa-2010-417-336	336	Nonno	grandfather	1383	GRANDFATHER	674	685	689	118	647	125	350
+DellaRosa-2010-417-337	337	Notte	night	1233	NIGHT	571	660	688	163	665	221	130
+DellaRosa-2010-417-338	338	Oca	goose	1187	GOOSE	685	700	468	150	659	106	197
+DellaRosa-2010-417-339	339	Oceano	ocean	645	OCEAN	594	642	555	295	594	165	396
+DellaRosa-2010-417-340	340	Ombrello	umbrella	3105	UMBRELLA	677	700	647	175	688	103	128
+DellaRosa-2010-417-341	341	Orologio	clock	1323	CLOCK	688	647	659	210	624	118	144
+DellaRosa-2010-417-342	342	Orso	bear	1820	BEAR	659	700	487	225	642	110	265
+DellaRosa-2010-417-343	343	Ospedale	hospital	616	HOSPITAL	658	693	656	241	696	167	270
+DellaRosa-2010-417-344	344	Ospite	guest	1782	GUEST	550	497	594	322	580	221	338
+DellaRosa-2010-417-345	345	Palma	palm			679	688	487	311	650	110	244
```

Comment:
This concept refers to PALM OF TREE, right? If so, this can me mapped to [PALM TREE 1181](https://concepticon.clld.org/parameters/1181). If not, I'm sure there is one concept for PALM (OF HAND) :)

Reply:
That's a tricky one and a nice colexification! Since we don't have any previous mappings and the "palm of hand" meaning seems dominant, I'd map to 1183 PALM OF HAND for now.